src/ArcgisItemCard/ArcgisItemCard.js

```diff
@@ -123,7 +128,9 @@ ArcgisItemCard.propTypes = {
   /** Style prop to position Card content vertically */
   vertical: PropTypes.bool,
   /** Whether the ArcgisItemCard shows an actions tab at the bottom or not */
-  actions: PropTypes.object
+  actions: PropTypes.object,
+  /** Whether the ArcgisItemCard should render the title differently */
+  customTitleRenderer: PropTypes.object
```

Comment:
I think the prop type would be `func`, no?

Reply:
That is correct. Thank you. 
Fixed here: 1b15c6b